MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -4084,6 +4105,29 @@ public void setCampaignOptions(CampaignOptions options) {
         campaignOptions = options;
     }
 
+    public StoryArc getStoryArc() { return storyArc; }
```

Comment:
I believe the style guide requires this to be 3 lines: https://github.com/MegaMek/megamek/wiki/MegaMek-Coding-Style-Guide#braces